packages/dashboard/src/pages/Settings/index.tsx

```diff
@@ -38,12 +38,12 @@ export default function Settings() {
                             desc={t.settings_appearance_desc()}>
                             <AppearanceSetting />
                         </SettingItem>
-                        <SettingItem
+                        {/* <SettingItem
```

Comment:
Why not just remove the node?

Reply:
Just hide this feature temporarily. Uncomment would be an easy way to put back.